src/webapp/pages/home/HomePage.tsx

```diff
@@ -17,9 +17,9 @@ import { Cardboard } from "../../components/card-board/Cardboard";
 import { BigCard } from "../../components/card-board/BigCard";
 import { goTo } from "../../utils/routes";
 import { defaultIcon, defaultTitle } from "../../router/Router";
-import { useAnalytics } from "../../hooks/useAnalytics";
 import { Maybe } from "../../../types/utils";
 import i18n from "../../../utils/i18n";
+import { CompositionRoot } from "../../CompositionRoot";
 
 export const HomePage: React.FC = React.memo(() => {
```

Comment:
The state management is growing moment by moment. Not something I would do in this PR, but I would add a TODO comment encouraging the next developer to extract state into a `useHomePage` hook.